go/processor/floria/processor_test.go

```diff
@@ -169,26 +169,31 @@ func TestProcessor_EoaCheck(t *testing.T) {
 }
 
 func TestProcessor_BuyGas(t *testing.T) {
-	balance := uint64(1000)
+	balance := uint64(1000000)
 	gasLimit := uint64(100)
 	gasPrice := uint64(2)
+	blobBaseFee := uint64(1)
 
 	transaction := tosca.Transaction{
-		Sender:   tosca.Address{1},
-		GasLimit: tosca.Gas(gasLimit),
+		Sender:     tosca.Address{1},
+		GasLimit:   tosca.Gas(gasLimit),
+		BlobHashes: []tosca.Hash{{0x01}},
 	}
 
+	newValue := balance - (gasLimit*gasPrice +
+		blobBaseFee*uint64(len(transaction.BlobHashes))*BlobTxBlobGasPerBlob)
+
 	ctrl := gomock.NewController(t)
 	context := tosca.NewMockTransactionContext(ctrl)
 	context.EXPECT().GetBalance(transaction.Sender).Return(tosca.NewValue(balance))
-	context.EXPECT().SetBalance(transaction.Sender, tosca.NewValue(balance-gasLimit*gasPrice))
-	context.EXPECT().GetBalance(transaction.Sender).Return(tosca.NewValue(balance - gasLimit*gasPrice))
+	context.EXPECT().SetBalance(transaction.Sender, tosca.NewValue(newValue))
+	context.EXPECT().GetBalance(transaction.Sender).Return(tosca.NewValue(newValue))
 
-	err := buyGas(transaction, context, tosca.NewValue(gasPrice))
+	err := buyGas(transaction, context, tosca.NewValue(gasPrice), tosca.NewValue(blobBaseFee))
 	if err != nil {
 		t.Errorf("buyGas returned an error: %v", err)
 	}
-	if context.GetBalance(transaction.Sender).Cmp(tosca.NewValue(balance-gasLimit*gasPrice)) != 0 {
+	if context.GetBalance(transaction.Sender).Cmp(tosca.NewValue(newValue)) != 0 {
```

Comment:
If I read this correctly, this line is only testing whether the expectation of line 190 is set up correctly. If so, it should be removed.